docs/sources/slack/example-api-responses/original/discovery-conversations-history.json

```diff
@@ -196,7 +196,107 @@
           ]
         }
       ]
+    },
+    {
+      "user": "USLACKBOT",
+      "channel": "D075J2RKK4Y",
+      "room": {
+        "id": "R07BNSQTZSP",
+        "name": "huddle test topic",
+        "media_server": "",
+        "created_by": "U02K5LRARED",
+        "date_start": 1720542372,
+        "date_end": 1720542756,
+        "participants": [],
```

Comment:
Redacted; no idea if it contains the plain list of ids like `participant_history` or the user objects like `parcitipants_events`